src/main/java/online/talkandtravel/controller/http/ChatController.java

```diff
@@ -51,6 +52,11 @@ public class ChatController {
 
   private final ChatService chatService;
 
+  @PostMapping
+  public ChatDto createCountrySubChat(@RequestBody NewSubChatRequest request) {
```

Comment:
lets remove word `sub` . All chats now are located in some country, and country main chats too. We have just chats, and they belong to some country. They are just chats, not subChats